packages/ai/core/embed/embed-many-result.ts

```diff
@@ -20,4 +20,22 @@ export interface EmbedManyResult<VALUE> {
   The embedding token usage.
     */
   readonly usage: EmbeddingModelUsage;
+
+  /**
+  Optional raw response data.
+     */
+  readonly rawResponses?: Array<
```

Comment:
Nice - should be eventually renamed to responses once the rename PR lands.

Reply:
is this similar to what we do for transcribe?

Reply:
Yep:

https://github.com/vercel/ai/blob/main/packages/ai/core/transcribe/transcribe-result.ts#L51C1-L54C65